src/Controller/Categorias/CategoriasOperarController.php

```diff
@@ -32,6 +38,18 @@ public function registrarEditarCategoria(CategoriaRepository $categoriasReposito
         $id = $request->request->get("id");
         $nome = $request->request->get("nome");
 
+        if(empty(trim($nome))){
+            $this->addFlash('danger', "O nome da categoria não pode estar vazio!");
+            return $this->redirectToRoute('editarCategoria', ['id' => $id]);
```

Comment:
ja cria um array com base na variavel, sugestão apenas
```suggestion
            return $this->redirectToRoute('editarCategoria', compact('id'));
```